website/source/docs/providers/nomad/r/job.html.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: "nomad"
+page_title: "Nomad: nomad_job"
+sidebar_current: "docs-nomad-resource-job"
+description: |-
+  Manages a job registered in Nomad.
+---
+
+# nomad\_job
+
+Manages a job registered in Nomad.
+
+This can be used to initialize your cluster with system jobs, common services,
+and more. In day to day Nomad use it is common for developers to submit
```

Comment:
The second sentence doesn't make sense to me? I think I'm either missing a Nomad concept or it's a little convoluted?